content/download.md

```diff
@@ -26,64 +26,67 @@ limitations under the License.
 
 ## {{ site.data.project.name }} Downloads
 
-There are no official Apache releases for {{ site.data.project.short_name }} yet.
-
-First Apache Incubator release coming soon!!
+The Official Apache release for {{ site.data.project.short_name }}.
 
 If you are interested in helping out, please join our developers list [{{site.data.project.dev_list}}](mailto:{{site.data.project.dev_list_subscribe}}?subject=subscribe to Rya dev list) by sending an email to [{{site.data.project.dev_list_subscribe}}](mailto:{{site.data.project.dev_list_subscribe}}?subject=subscribe to Rya dev list).
 
-{% comment %}
-
-TODO: uncoment this when we have official releases of Rya
-
 {{ site.data.project.name }} is released as a source artifact, and also through Maven.
 
 ### Source releases
 
-Release          | Date       | Commit   | Download
-:--------------- | :--------- | :------- | :-------
+Release          | Date       | Commit   | Download | Notes
+:--------------- | :--------- | :------- | :------- | :----
 {% for post in site.categories.release %}{% comment %}
 {% endcomment %}{% if post.fullVersion %}{% comment %}
 {% endcomment %}{% assign v = post.fullVersion %}{% comment %}
 {% endcomment %}{% else %}{% comment %}
 {% endcomment %}{% capture v %}apache-{{ site.data.project.unix_name }}-{{ post.version }}{% endcapture %}{% comment %}
 {% endcomment %}{% endif %}{% comment %}
 {% endcomment %}{% if forloop.index0 < 2 %}{% comment %}
-{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua?filename={{ site.data.project.unix_name }}/{{ v }}{% endcapture %}{% comment %}
+{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua?filename={{ site.data.project.incubator_slash_name }}/{{ v }}{% endcapture %}{% comment %}
```

Comment:
The link (http://www.apache.org/dyn/closer.lua?filename=incubator/rya/rya-incubating-3.2.10/rya-project-3.2.10-incubating-source-release&action=download) did not work well for me. The generated URL should be http://www.apache.org/dyn/closer.lua/incubator/rya/rya-incubating-3.2.10/rya-project-3.2.10-incubating-source-release.zip